libs/api/domains/documents/src/lib/document.service.ts

```diff
@@ -154,4 +156,43 @@ export class DocumentService {
       return []
     }
   }
+
+  async getPaperMailInfo(nationalId: string): Promise<PaperMailBody> {
+    try {
+      const res = await this.documentClient.requestPaperMail(nationalId)
+      return {
+        nationalId: res?.kennitala,
+        wantsPaper: res?.wantsPaper,
+      }
+    } catch (exception) {
+      logger.error(exception)
+      return {
+        nationalId,
+        wantsPaper: undefined,
+      }
+    }
+  }
+
+  async postPaperMailInfo(
+    nationalId: string,
+    body: PostRequestPaperInput,
+  ): Promise<PaperMailBody> {
+    try {
+      const postBody = {
+        kennitala: nationalId,
+        wantsPaper: body.wantsPaper,
+      }
+      const res = await this.documentClient.postPaperMail(postBody)
+      return {
+        nationalId: res?.kennitala,
+        wantsPaper: res?.wantsPaper,
+      }
+    } catch (exception) {
+      logger.error(exception)
```

Comment:
see above

Reply:
This is just a continuation of the previous document service functions. Will re-visit on re-write